pkg/config/virtio_test.go

```diff
@@ -296,6 +320,12 @@ func testErrorVirtioDev(t *testing.T, test *virtioDevTest) {
 }
 
 func TestVirtioDevices(t *testing.T) {
+	defer func() {
+		err := os.Remove(testImagePath)
+		if err != nil {
+			log.Warn("Failed to remove test image")
+		}
+	}()
```

Comment:
This can replaced with:

    testImagePath := filepath.Join(t.TempDir(), "test.img")